config/app.php

```diff
@@ -196,19 +196,33 @@
     'private_uploads' => storage_path().'/private_uploads',
 
 
+    /*
+   |--------------------------------------------------------------------------
+   | ALLOW I-FRAMING
+   |--------------------------------------------------------------------------
+   |
+   | Normal users will never need to edit this. This option lets you run
+   | Snipe-IT within an I-Frame, which is normally disabled by default for
+   | security reasons, to prevent clickjacking. It should normally be set to false.
+   |
+   */
+
+    'allow_iframing' => env('ALLOW_IFRAMING', false),
+
+
     /*
     |--------------------------------------------------------------------------
-    | ALLOW I-FRAMING
+    | ENABLE HTTP Strict Transport Security (HSTS)
```

Comment:
I know it sounds horrifically stupid, but I think if you re-order your commands here, this diff will be smaller. And I think that's good because there's less that's _visibly_ changing.

Reply:
Nope, I already tried that. Git is getting confused for some reason, but this *is* the reordered version. :-/